reporters/kamon-prometheus/src/main/resources/reference.conf

```diff
@@ -58,6 +58,26 @@ kamon.prometheus {
     }
   }
 
+  summary: {
+    # Determines if summaries should be added in addition to or instead of prometheus-histogram
+    exclusive: false
```

Comment:
I just found this on the [Prometheus docs](https://prometheus.io/docs/instrumenting/exposition_formats/#comments-help-text-and-type-information):

> Only one TYPE line may exist for a given metric name.

So, a metric should be reported as a summary or a histogram, but never both. We should remove this setting and ensure that all metrics selected to be reported as summaries are only reported as summaries.

Reply:
you're right. removed that config value and made it exclusive for every summary